script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3.txt

```diff
@@ -15,7 +15,7 @@ In the end, I went to the staff room and found where it was on the floor plan.
 <Page5>
 "Excuse me. Senpai, are you there?"
 I knock on the sliding door and call out to her.//横開きの扉をノックして、呼びかける。
-Soon after.
+Soon after, //Matches JP text and flows better
```

Comment:
I think we've tried to avoid dangling commas on page breaks, either converting them to full stops or three CJK dashes to indicate the truncation